nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/ProcessorResource.java

```diff
@@ -489,7 +489,7 @@ public Response getState(
      * @throws InterruptedException if interrupted
      */
     @POST
-    @Consumes(MediaType.WILDCARD)
+    @Consumes(MediaType.APPLICATION_JSON)
```

Comment:
Following the pattern of other Resource methods, it looks like this should support both wildcard and JSON, correct?